metricbeat/docs/modules/mssql.asciidoc

```diff
@@ -30,6 +30,31 @@ The following Metricsets are already included:
 
 `performance` Metricset fetches information from what's commonly known as https://docs.microsoft.com/en-us/sql/relational-databases/system-dynamic-management-views/sys-dm-os-performance-counters-transact-sql?view=sql-server-2017[Performance Counters] in MSSQL.
 
+=== Module-specific configuration notes
```

Comment:
Unrelated to CI error, but you probably want a `[float]` tag here before this section (otherwise this content will become a new topic).